node/pkg/governor/mainnet_chains_test.go

```diff
@@ -19,32 +20,40 @@ func TestChainListSize(t *testing.T) {
 func TestChainDailyLimitRange(t *testing.T) {
 	chainConfigEntries := chainList()
 
+	/* These IS a hard limit, if daily limit is set to zero it would
+	   basically mean no value moment is allowed for that chain*/
+	min_daily_limit := uint64(0)
+
+	// Do not remove this assertion
+	assert.Greater(t, min_daily_limit, uint64(0))
```

Comment:
`min_daily_limit` is 0 so doesn't this fail every time?  Is this supposed to be for `max_daily_limit`?